README.md

```diff
@@ -1,211 +1,119 @@
-
-- [Virtualbox](https://www.virtualbox.org), the Virtual Machine runner that Minikube
-will use to run Kubernetes.
-
-
-
-### Deploying on Minikube
-
-Follow each of these steps in your terminal to get a local Studio deployment
-in a cluster on your local machine:
-
-  1. Clone the Studio repo to your local machine:
-
-
-  ```bash
-  $ git clone https://github.com/learningequality/studio
-  $ cd studio/
-  ```
-
-  1. Go to the `k8s/` directory in the Kolibri studio repo:
+[Beta Site](https://studio.learningequality.org)
 
-  ```bash
-  $ cd k8s/
-  ```
-
-  1. Get your local Kubernetes cluster up and running using Minikube:
-
-  ```bash
-  $ minikube start --kubernetes-version v1.8.0
-  ```
-
-  This should start the Kubernetes cluster, and set up kubectl to
-  refer to the minikube cluster.
-
-  1. Point the docker command to the docker daemon running in Minikube:
-
-  ```bash
-  $ eval $(minikube docker-env)
-  ```
-
-  1. Build the docker images:
-
-  ```bash
-  $ make build
-  ```
-
-  1. Initialize your Kubernetes cluster with Helm, to allow it to deploy Kubernetes manifests:
-
-  ```
-  $ helm init
-  ```
-
-  1. Use the images to deploy Kolibri Studio:
-
-  ```
-  $ helm upgrade --install mystudio .
-  ```
-
-  1. Once Studio is ready to use (it should take about a minute), run this command:
-
-  ```
-  $ minikube service mystudio-studio-app --url
-  ```
-
-  and open the IP address displayed, in to your browser. You should now see the Studio login page!
-
-
-
-## Developer instructions
+Kolibri Studio is a web application designed to deliver educational materials to [Kolibri](http://learningequality.org/kolibri/).
+Kolibri Studio supports the following workflows:
 
-Follow the instructions below to setup your dev environment and get started.
+- Organizing and publishing content channels in the format suitable for import from Kolibri.
 
+- Curating content and remixing of existing channels into custom channels aligned to various educational standards, country curricula, and special needs.
 
-### Get the code
+- Creating learning pathways and assessments.
 
-  - Fork the [studio repo](https://github.com/learningequality/studio) to create
-    a copy of the studio repository under your own username on github.
-    Note: the code examples below assume your username is `yourusername`, please
-    modify and replace with your own user name before running the commands.
+- Uploading new content through the web interface or programatically using [ricecooker-powered](https://github.com/learningequality/ricecooker) chef scripts.
 
-  - Clone your fork of the repository to your local machine:
+Kolibri Studio uses Django for the backend and Backbone.js for the frontend.
 
-        cd MyCodingProjectsDir
-        git clone git@github.com:yourusername/studio.git
+If you would like to get a local preview of Studio, please follow [these instructions](https://github.com/learningequality/studio/blob/develop/docs-developer/local_preview.md)
```

Comment:
I think we should avoid absolute links like this, as master's README.md will point to the develop branch's docs. I kind of feel like we should be moving anything not part of the core setup to RTD and pointing people there.